.github/workflows/release.yml

```diff
@@ -87,6 +87,66 @@ jobs:
             kup publish k-framework-binary .#k --keep-days 180
             kup publish k-framework-binary .#k.openssl.procps.secp256k1 --keep-days 180
 
+  ubuntu-noble:
+    name: 'K Ubuntu Noble Package'
+    runs-on: [self-hosted, linux, normal]
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v4
+      - name: 'Build and Test Package'
+        uses: ./.github/actions/test-package
+        with:
+          os: ubuntu
+          distro: noble
+          llvm: 16
+          pkg-name: kframework_amd64_ubuntu_noble.deb
+          build-package: package/debian/build-package noble kframework
+          test-package: package/debian/test-package
+      - name: 'Upload the package built to the Summary Page'
+        uses: actions/upload-artifact@v4
+        with:
+          name: kframework_amd64_ubuntu_noble.deb
+          path: kframework_amd64_ubuntu_noble.deb
+          if-no-files-found: error
+          retention-days: 1
+      - name: 'Upload Package to Release'
+        env:
+          GITHUB_TOKEN: ${{ secrets.JENKINS_GITHUB_PAT }}
+        run: |
+          set -x
+          version=$(cat package/version)
+          cp kframework_amd64_ubuntu_noble.deb "kframework_${version}_amd64_ubuntu_noble.deb"
+          gh release upload --repo runtimeverification/k --clobber "v${version}" "kframework_${version}_amd64_ubuntu_noble.deb"
+      - name: 'Build, Test, and Push Dockerhub Image'
+        shell: bash {0}
+        env:
+          DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
+          DOCKERHUB_REPO: runtimeverificationinc/kframework-k
+        run: |
+          set -euxo pipefail
+          version=$(cat package/version)
+          version_tag=ubuntu-noble-${version}
+          docker login --username rvdockerhub --password "${DOCKERHUB_PASSWORD}"
+          docker image build . --file package/docker/Dockerfile.ubuntu-noble --tag "${DOCKERHUB_REPO}:${version_tag}"
```

Comment:
`Dockerfile.ubuntu-noble` seems to be missing.

Reply:
Ah, thank you.